### DISCUSSION TITLE
突然dashboard打开变空白，重启无效

### DISCUSSION THREAD

**Discussion by YOU-Y8** [vernesong/OpenClash#2654] [Q&A]
![屏幕截图 2022-08-25 001518](https://user-images.githubusercontent.com/73763161/186470327-404fcb93-811a-45e9-9822-c0b32f2a4030.jpg)
按f12显示红色那三项都是404，是因为丢了什么文件才打不开吗？

**Comment by vernesong** [comment]
ctrl f5
